apps/testkit.net/Testkit.Net.Tests/Longevity.cs

```diff
@@ -42,79 +44,79 @@ public Longevity(string syncGatewayUrl, double scenarioRuntimeMinutes)
-                Database = _db,
-                Target = new ReplicatorTarget(new Uri($"{syncGatewayUrl}")),
                 Continuous = true,
                 ReplicatorType = ReplicatorType.PushAndPull
-            };
-            _replicator = new Replicator(replicatorConfig);
-            _replicator.StatusChanged += (sender, args) =>
-            {
-                Console.WriteLine($"Replication Activity: {args.Status.Activity}");
-                Console.WriteLine($"Replication Completed: {args.Status.Progress.Completed}");
-                Console.WriteLine($"Replication Total: {args.Status.Progress.Total}");
-            };
+            };
+
+            _replicator = new Replicator(replicatorConfig);
+            _replicator.StatusChanged += (sender, args) =>
+                {
+                    Console.WriteLine($"Replication Activity: {args.Status.Activity}");
+                    Console.WriteLine($"Replication Completed: {args.Status.Progress.Completed}");
+                    Console.WriteLine($"Replication Total: {args.Status.Progress.Total}");
+                };
             _replicator.Start();
-        }
-
-        [Fact]
-        public void Run()
+        }
+
+        [Fact]
+        public void Run()
         {
             // Create docs
-            var numDocs = 100000;
-            Console.WriteLine($"Saving: {numDocs} docs");
-            _db.InBatch(() =>
-            {
-                for (int i = 0; i < numDocs; i++)
-                {
-                    var doc = new Document($"doc_{i}");
-                    doc["random"].Value = Guid.NewGuid().ToString();
-                    _db.Save(doc);
-                }
-            });
-
-            // Update docs
-            var stopWatch = new Stopwatch();
-            stopWatch.Start();
-            var r = new Random();
-            while (true)
-            {
-                double totalMin = stopWatch.Elapsed.TotalMinutes;
-
-
-                if (totalMin > _scenarioRuntimeMinutes)
-                {
-                    // We have reached the total time we would like to run
-                    // for. Exit
-                    break;
-                }
-
-                // Update a random doc
-                int rInt = r.Next(0, numDocs);
-                Document doc = _db.GetDocument($"doc_{rInt}");
-
-                Console.WriteLine($"Updating doc: {doc.Id}");
-
-                doc["random"].Value = Guid.NewGuid().ToString();
-                _db.Save(doc);
-
-                // Sleep between each update
-                Task.Delay(100).Wait();
-            }
-
-            // Delete docs
-            Console.WriteLine($"Deleting: {numDocs} docs");
-            for (int i = 0; i < numDocs; i++)
-            {
-                var doc = _db.GetDocument($"doc_{i}");
-                _db.Delete(doc);
-            }
-
-            stopWatch.Stop();
-            _replicator.Stop();
-            _db.Delete();
+            var numDocs = _maxDocs;

```

Comment:
Is it necessary to create another variable numDocs, can we use _maxDocs directly here